AsyncDisplayKit/Layout/ASRelativeLayoutSpec.h

```diff
@@ -15,21 +15,21 @@
 /** How the child is positioned within the spec. */
 typedef NS_OPTIONS(NSUInteger, ASRelativeLayoutSpecPosition) {
     /** The child is positioned at point 0 relatively to the layout axis (ie left / top most) */
-    ASRelativeLayoutSpecPositionStart = 0,
+    ASRelativeLayoutSpecPositionStart = 1 << 0,
     /** The child is centered along the specified axis */
-    ASRelativeLayoutSpecPositionCenter = 1 << 0,
+    ASRelativeLayoutSpecPositionCenter = 1 << 1,
     /** The child is positioned at the maximum point of the layout axis (ie right / bottom most) */
-    ASRelativeLayoutSpecPositionEnd = 1 << 1,
+    ASRelativeLayoutSpecPositionEnd = 1 << 2,
 };
 
 /** How much space the spec will take up. */
 typedef NS_OPTIONS(NSUInteger, ASRelativeLayoutSpecSizingOption) {
     /** The spec will take up the maximum size possible */
-    ASRelativeLayoutSpecSizingOptionDefault,
+    ASRelativeLayoutSpecSizingOptionDefault = 1 << 0,
```

Comment:
Discussing this more, changing the default value here to 1 creates an undefined case for Swift users. In Swift, it should always be valid to provide `[]` for an `OptionSet`. If they do that here, then we have an undefined value.

I think the best thing we can do is to restore the original values in here, and add documentation that in Swift, pass `[]` to get `ASRelativeLayoutSpecSizingOptionDefault`. Apple has made exceptions in their own API to import 0-value options (e.g. `UIViewAutoResizing`) but we're not so lucky…